src/features/landing/components/AboutVilnaSection/AboutVilnaDesktopNotchSvg/AboutVilnaDesktopNotchSvg.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+
+const styles = {
+  position: 'absolute',
+  zIndex: '950',
+  top: '16px',
+  left: '347px',
+};
+
+export default function AboutVilnaDesktopNotchSvg({ style }: { style?: React.CSSProperties }) {
+  return (
+    <svg
```

Comment:
need refactor all of this, MUI have SvgIcon component, and you can change styles for SVG from CSS